packages/rancher-kiali-server-1.35/package.yaml

```diff
@@ -0,0 +1,8 @@
+url: https://kiali.org/helm-charts/kiali-server-1.35.0.tgz
```

Comment:
Why'd we make this package (the folder) version specific instead of reusing the standard one ? I'm assuming its related to my previous question, you have to introduce two kiali versions. But even if you do, is 1 kiali version not compatible across 1.9 and 1.10 istio ?

Reply:
We are not making chart changes to 1.8. or 1.9 in this release, except for moving them to the new chart scripts versioning schema. Also, with the refresh issues in kiali 1.35, it would be preferable to leave istio 1.8 and 1.9 on the current kiali version so that we aren't introducing that issue to more istio versions than we have to.